edgelet/edgelet-kube/tests/runtime.rs

```diff
@@ -142,7 +142,7 @@ fn authenticate_returns_none_when_no_auth_token_provided() {
     runtime.spawn(server);
     let auth_id = runtime.block_on(task).unwrap();
 
-    assert_eq!(AuthId::None, auth_id)
+    assert_eq!(auth_id, AuthId::None)
```

Comment:
Why did you flip these? Isn't it more readable to have the expected result on the left?

Reply:
I added these asserts earlier and misplaced them. Now I just fix those mistypes.
Intellij Idea conveniently converts `left`/`right` params to `actual`/`expected` and wants `expected` to be on the right side of assert.

Reply:
"Conveniently" ;)